src/components/site-form.tsx

```diff
@@ -106,29 +223,77 @@ export const SiteForm = ( {
 	doesPathContainWordPress?: boolean;
 	isPathInputDisabled?: boolean;
 	onSubmit: ( event: FormEvent ) => void;
+	fileForImport?: File | null;
+	setFileForImport?: ( file: File | null ) => void;
+	onFileSelected?: ( file: File ) => void;
+	fileError?: string;
 } ) => {
 	const { __ } = useI18n();
 
+	const [ isAdvancedSettingsVisible, setAdvancedSettingsVisible ] = useState( false );
+
+	const handleAdvancedSettingsClick = () => {
+		setAdvancedSettingsVisible( ! isAdvancedSettingsVisible );
+	};
+
 	return (
 		<form className={ className } onSubmit={ onSubmit }>
 			<div className="flex flex-col gap-6">
 				<label className="flex flex-col gap-1.5 leading-4">
 					<span className="font-semibold">{ __( 'Site name' ) }</span>
 					<TextControlComponent onChange={ setSiteName } value={ siteName }></TextControlComponent>
 				</label>
-				{ onSelectPath && (
-					<label className="flex flex-col gap-1.5 leading-4">
-						<span onClick={ onSelectPath } className="font-semibold">
-							{ __( 'Local path' ) }
-						</span>
-						<FormPathInputComponent
-							isDisabled={ isPathInputDisabled }
-							doesPathContainWordPress={ doesPathContainWordPress }
-							error={ error }
-							value={ sitePath }
-							onClick={ onSelectPath }
-						/>
-					</label>
+				{ setFileForImport && (
+					<>
+						<div className="flex flex-col gap-1.5 leading-4">
+							<label className="font-semibold">
+								{ __( 'Import a backup' ) }
+								<span className="font-normal">{ __( ' (optional)' ) }</span>
+							</label>
+							<span className="text-a8c-gray-50 text-xs">
+								{ createInterpolateElement(
+									__( 'Jetpack and WordPress backups supported. <button>Learn more</button>' ),
+									{
+										button: (
+											<Button
+												variant="link"
+												className="text-xs"
+												onClick={ () => getIpcApi().openURL( STUDIO_DOCS_URL ) }
+											/>
+										),
+									}
+								) }
+							</span>
+							<FormImportComponent
+								placeholder={ __( 'Select or drop a file' ) }
+								value={ fileForImport }
+								onChange={ ( file ) => setFileForImport( file ) }
+								onClear={ () => setFileForImport( null ) }
+								onFileSelected={ onFileSelected }
+								error={ fileError }
+							/>
+						</div>
+						<div className="flex flex-row">
+							<Button className="pl-0" onClick={ handleAdvancedSettingsClick }>
+								<Icon size={ 24 } icon={ isAdvancedSettingsVisible ? chevronDown : chevronRight } />
+								<div className="text-[13px] leading-[16px] ml-2">{ __( 'Advanced settings' ) }</div>
+							</Button>
+						</div>
+						{ onSelectPath && isAdvancedSettingsVisible && (
```

Comment:
Should this `onSelectPath` check be wrapping around the "advanced settings" ? I want to avoid the case where we forget to pass `onSelectPath` and then we have an empty expandable "advanced view"
```suggestion
						{ isAdvancedSettingsVisible && (
```

Reply:
Not a strong opinion, since we this behaviour will change and make more sense when we add the WP and PHP versions inside the advanced view.

Reply:
you are right, good catch. I updated this in https://github.com/Automattic/studio/pull/378/commits/8e36959b9173999606b0d4f43acf5ff3facf5f1d

I think we will end up refactoring this in the future probably anyway but it is a good idea to take care of this at the moment